### PR TITLE
feat: Add arguments for model parameters

### PR DESCRIPTION
**Pull request by carlosgjs**
- Expose `temperature, top_k and top_n` parameters for both eval and generation pipelines (function -> CLI -> .yml workflow)
- Add a prompt with a "one-shot" example
- Add a notebook for testing generation

### CODE REVIEW

File: README.md
Comment (lsetiawan) [inline README.md:1]: Currently PyPi and docs page doesn't lead anywhere, while you're modifying this file, I suggest commenting out these badges for now.

File: src/autora/doc/pipelines/main.py [hunk 1]
```diff
@@ -16,13 +16,24 @@
 logger = logging.getLogger(__name__)
 
 
-@app.command()
-def eval(data_file: str, model_path: str, sys_id: SystemPrompts, instruc_id: InstructionPrompts) -> List[str]:
+@app.command(help="Evaluate model on a data file")
+def eval(
+    data_file: str = typer.Argument(..., help="JSONL Data file to evaluate on"),
+    model_path: str = typer.Option("meta-llama/Llama-2-7b-chat-hf", help="Path to HF model"),
```
Comment (lsetiawan) [inline src/autora/doc/pipelines/main.py:22]: Does this model already exist? If not, this might be better if it's a required argument?

Comment (carlosgjs) [reply src/autora/doc/pipelines/main.py:22]: It does exist and will be downloaded on-demand from huggingface on first use if it's not already in the cache:

https://huggingface.co/meta-llama/Llama-2-7b-chat-hf

Comment (lsetiawan) [reply src/autora/doc/pipelines/main.py:22]: Gotcha, alrighty. If it will exist in file by default, then this is okay. Thanks!

**Review by lsetiawan — APPROVED** [review]
This PR looks good to me. I can't fully comment on actual mlflow implementation however. If all works, I believe it's good 😬

**Pull request merged**